cmake/configure_files/wcs_config.hpp.in

```diff
@@ -5,6 +5,19 @@
 #define WCS_VERSION @WCS_VERSION@
 #cmakedefine WCS_GIT_VERSION @WCS_GIT_VERSION@
 
+/* Defined for the generation of the c++ code of a sbml file */
+#cmakedefine CMAKE_CXX_COMPILER "@CMAKE_CXX_COMPILER@"
+#ifndef CMAKE_CXX_COMPILER
+#define CMAKE_CXX_COMPILER "cc"
+#endif
+
+#define CMAKE_CXX_FLAGS "@CMAKE_CXX_FLAGS@"
+
+#cmakedefine CMAKE_CXX_SHARED_LIBRARY_FLAGS "@CMAKE_CXX_SHARED_LIBRARY_FLAGS@"
```

Comment:
```#define CMAKE_CXX_SHARED_LIBRARY_FLAGS "@CMAKE_CXX_SHARED_LIBRARY_FLAGS@"```
Then, remove the three lines below, 17-19.
If it was working for CMAKE_CXX_FLAGS, it should be ok for this as well.